src/hotspot/cpu/ppc/frame_ppc.cpp

```diff
@@ -305,7 +305,7 @@ BasicType frame::interpreter_frame_result(oop* oop_result, jvalue* value_result)
       case T_OBJECT:
       case T_ARRAY: {
         oop obj = *(oop*)tos_addr;
-        assert(obj == NULL || Universe::heap()->is_in(obj), "sanity check");
+        assert(obj == NULL || Universe::is_in_heap(obj), "sanity check");
```

Comment:
Could have been changed to is_in_heap_or_null.

Reply:
Fixed. I also change other frame_<cpu>.cpp files to use is_in_heap_or_null.